doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -301,8 +301,7 @@ Ensure you have an MDS running and issue:
 .. note::
 
    Symbolic links are recovered as empty regular files. `Symbolic link recovery
```

Comment:
Do we need this line at all?

```
Symbolic links are recovered as empty regular files.
```

We do maintain per release docs.

Reply:
I think it makes things clear. If we remove it, it might indicate symlinks were not recovered at all? I think we should make it past tense though. What do you think ?

Reply:
Yeh, I meant the same. Advertise that it works first :)

Reply:
Done